helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/TestAssignableReplica.java

```diff
@@ -96,4 +101,29 @@ public void testConstructRepliaWithResourceConfig() throws IOException {
     Assert.assertEquals(replica.getResourceInstanceGroupTag(), group);
     Assert.assertEquals(replica.getResourceMaxPartitionsPerInstance(), maxPartition);
   }
+
+  @Test
+  public void testIncompletePartitionWeightConfig() throws IOException {
```

Comment:
Question:

If we don't have the key, are we treating it as 0 or erroring out?

Reply:
For replica, it would be 0.